libraries/AP_GPS/AP_GPS_SBP2.cpp

```diff
@@ -333,6 +333,7 @@ AP_GPS_SBP2::_attempt_state_update()
             case 1:
                 state.status = AP_GPS::GPS_OK_FIX_3D;
                 break;
+            case 6:
```

Comment:
This adds the SBAS fix as a 'DGPS' 3D fix.  It might also be appropriate to add it with case 1 above (as just a regular 3D fix) instead?